src/Engine/ProtoCore/Lang/Type.cs

```diff
@@ -121,6 +121,16 @@ public class TypeSystem
         public Dictionary<ProtoCore.DSASM.AddressType, int> addressTypeClassMap { get; set; }
         private static Dictionary<PrimitiveType, string> primitiveTypeNames;
 
+        private static readonly Dictionary<string, string> clrToDSTypeMap = new Dictionary<string, string>
+        {
+            { $"{nameof(System)}.{nameof(String)}", "string" },
+            { "long", "int" },
```

Comment:
where does the `long` type name come from given that `int64` is the actual .net type name?

Reply:
considering that this dictionary is only created once - maybe it's not so bad to do:
`typeof(Int64).FullName`

Reply:
I was thinking that someone could manually type `long`, but that's highly unlikely.

Reply:
I can remove it.